File: webapp/src/main/webapp/WEB-INF/cliente/app/model/contabilidad/asientoFiltro.ts
export class AsientoFiltro{
  constructor(
      public id:string,
      public descripcion:string,
      public cuenta:string,
      public concepto:string){}
}

export class ResumenAsiento{
  constructor(
      public id:number,
      public descripcion:string,
      public saldo:number
  ){}
}

File: service/src/main/java/org/dbp/dto/contabilidad/AsientoFiltro.java
package org.dbp.dto.contabilidad;

import java.io.Serializable;

public class AsientoFiltro implements Serializable{

	private String id;
	private String descripcion;
	private String cuenta;
	private String concepto;
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getDescripcion() {
		return descripcion;
	}
	public void setDescripcion(String descipcion) {
		this.descripcion = descipcion;
	}
	public String getCuenta() {
		return cuenta;
	}
	public void setCuenta(String cuenta) {
		this.cuenta = cuenta;
	}
	public String getConcepto() {
		return concepto;
	}
	public void setConcepto(String concepto) {
		this.concepto = concepto;
	}
	
	
	
}
